_posts/2015-03-14-rqlite---building-SQLite-clusters.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: rqlite - building SQLite clusters
```

Comment:
What about "Building SQLite clusters with rqlite"? It looks better for a blog post title, doesn't it?